src/bindings/jna/libelektra5j/src/main/java/org/libelektra/Elektra.java

```diff
@@ -146,22 +146,17 @@ int elektraGOptsContractFromStrings (Pointer contract, long argsSize, String arg
 
 	Pointer ksCut (Pointer ks, Pointer cutpoint);
 
+	// TODO #3137 Also elektraKsPopAtCursor should replace the current ksPop. See also #3189.
```

Comment:
I think you should use `elektraKsPopAtCursor` instead of `ksPop` here and then update `KeySet::pop`. Also `KeySet::pop` should maybe be renamed to `KeySet::remove` (I think that is a more common name for this kind of operation in Java).

Then we just need to change the names, but not any code when we update the C function.

Reply:
done